media/videostitcher/update_slate.go

```diff
@@ -53,10 +53,10 @@ func updateSlate(w io.Writer, projectID, location, slateID, slateURI string) err
 	// Updates the slate.
 	response, err := client.UpdateSlate(ctx, req)
 	if err != nil {
-		return fmt.Errorf("UpdateSlate: %v", err)
+		return fmt.Errorf("client.UpdateSlate: %v", err)
 	}
 
-	fmt.Fprintf(w, "Updated slate: %v", response.Name)
+	fmt.Fprintf(w, "Updated slate: %v", response)
```

Comment:
If you'd like to dump the whole response, use `%+v` formatter to include both keys and values.

Reply:
Thanks! Added formatter